interfaces/builtin/unity7.go

```diff
@@ -327,7 +327,7 @@ dbus (send)
     path=/org/freedesktop/Notifications
     interface=org.freedesktop.Notifications
     member="{GetCapabilities,GetServerInformation,Notify}"
-    peer=(name=org.freedesktop.Notifications, label=unconfined),
```

Comment:
This is the denial:
```
Jul 26 09:23:42 foo dbus[4229]: apparmor="DENIED" operation="dbus_method_call"  bus="session" path="/org/freedesktop/Notifications" interface="org.freedesktop.Notifications" member="GetServerInformation" mask="send" name=":1.325" pid=12970 label="snap.teatime.teatime" peer_pid=8575 peer_label="unconfined"
```

Before we were only allowing calling GetCapabilities, GetServerInformation and Notify if we sent to the well-known name of "org.freedesktop.Notifications", but that was too strict since the teatime timer needed to send to a non-well-known name, in this case, ":1.325". The only reason we had "name=org.freedesktop.Notifications" was that prior testing showed we could be that specific, but teatime demonstrated we cannot (note there are a few other rules that don't specify the peer name; we care mostly about the peer label which remains unchanged).